source/solvers/multiphysics_interface.cc

```diff
@@ -41,5 +41,65 @@ MultiphysicsInterface<dim>::MultiphysicsInterface(
     }
 }
 
+template <int dim>
+TrilinosWrappers::MPI::Vector *
+MultiphysicsInterface<dim>::get_filtered_phase_fraction_gradient_solution()
+{
+  AssertThrow((std::find(active_physics.begin(),
```

Comment:
For clarity, add a comment before that, like
`Return error if VOF solver is not enabled` or `VOF solver needs to be enabled`

Reply:
done.